src/plugins/Workers/IdentitySyncWorker.js

```diff
@@ -67,7 +67,7 @@ class IdentitySyncWorker extends Worker {
 
       // if identity id is not preset then increment gap count
       // and stop sync if gap limit is reached
-      if (fetchedId === null) {
+      if (fetchedId == null) {
```

Comment:
It supposed to be `null`, since DAPI should respond with an array with one element.